build/config/compiler/compiler.gni

```diff
@@ -21,7 +21,8 @@ declare_args() {
 
   # Optimize debug builds with -Og.
   optimize_debug = current_os == "freertos" || current_os == "zephyr" ||
-                   current_os == "mbed" || current_os == "cmsis-rtos"
+                   current_os == "mbed" || current_os == "cmsis-rtos" ||
+                   current_os == "mac" || current_os == "ios"
```

Comment:
Why this change?  `-Og` is decent, but it still makes debugging a bit of a pain compared to not optimizing at all...

Reply:
I've removed that commit from the PR now.